src/logdata/src/logdata.cpp

```diff
@@ -419,8 +419,8 @@ std::vector<QString> LogData::getLinesFromFile( LineNumber firstLine, LinesCount
 
         processedLines.reserve( decodedLines.size() );
 
-        for ( auto&& line : decodedLines ) {
-            processedLines.push_back( processLine( std::move( line ) ) );
+        for ( const auto& line : decodedLines ) {
+            processedLines.emplace_back( line );
```

Comment:
Before pushing result back the line needs to be passed to `processLine` callable argument

Reply:
@variar, fix it

Reply:
@GermanAizek I don't see any new commits in PR. Did you push the fix?

Reply:
@variar, I made a force push of the last commit by overwriting it